core/components/css-utilities/Miscellaneous/Miscellaneous.story.tsx

```diff
@@ -24,20 +24,631 @@ export const miscellaneous = () => {
-        />
-      </Card>
+    <div className="p-4">
+      <Heading size="xxl">Miscellaneous Utilities</Heading>
+      <br />
+      <Text appearance="default">
+        A collection of utility classes that provide various styles and behaviors not covered by other categories.
+      </Text>
+      <div className="mt-8 mb-8">
+        <Card className="h-100">
+          <Table
+            data={data}
+            schema={utilitiesSchema}
+            headerOptions={{
+              withSearch: true,
+            }}
+            size={'standard'}
+            showMenu={false}
+          />
+        </Card>
+      </div>
+
+      {/* Examples section */}
+      <Heading size="m" className="mt-8">
+        Examples
+      </Heading>
+      <Text appearance="default">Here are some examples of these utility classes in action:</Text>
+
+      {/* User Select example */}
+      <div className="mt-8 mb-8">
+        <Heading size="s" className="mb-3">
+          User Select
+        </Heading>
+        <div className="d-flex flex-wrap gap-4">
+          <div className="w-25 mb-3 user-select-none">
+            <Card className="p-4 bg-secondary-lightest shadow-s">
+              <Heading appearance="subtle" size="s" className="color-primary-dark mb-2">
+                user-select-none
+              </Heading>
+              <div className="bg-white p-3 rounded-10 border border-primary">
+                Try to <span className="font-weight-bold">select this text</span> - it cannot be selected
+              </div>
+            </Card>
+          </div>
+          <div className="w-25 mb-3 user-select-all">
+            <Card className="p-4 bg-secondary-lightest shadow-s">
+              <Heading appearance="subtle" size="s" className="color-accent3-dark mb-2">
+                user-select-all
+              </Heading>
+              <div className="bg-white p-3 rounded-10 border border-accent3">
+                Click anywhere to <span className="font-weight-bold">select all of this text</span> at once
+              </div>
+            </Card>
+          </div>
+          <div className="w-25 mb-3 user-select-text">
+            <Card className="p-4 bg-secondary-lightest shadow-s">
+              <Heading appearance="subtle" size="s" className="color-success-dark mb-2">
+                user-select-text
+              </Heading>
+              <div className="bg-white p-3 rounded-10 border border-success">
+                This has <span className="font-weight-bold">normal text selection</span> behavior (default)
+              </div>
+            </Card>
+          </div>
+        </div>
+      </div>
+
+      {/* Word Break example */}
+      <div className="mt-8 mb-8">
+        <Heading size="s" className="mb-3">
+          Word Break
+        </Heading>
+        <div className="d-flex flex-wrap gap-4">
+          <div className="w-25 mb-5">
+            <div className="text-center mb-2 color-primary-dark font-weight-bold">word-break-normal</div>
```

Comment:
classname `font-weight-bold` does not exist in our design system. Please remove it from everywhere.